docs/swagger.yml

```diff
@@ -1377,9 +1377,256 @@ definitions:
+      cpu_socket_count:
+        type: "integer"
+        format: "int64"
+        description: "The count of cpu sockets"
+        example: 2
+      cpu_core_count:
+        type: "integer"
+        format: "int64"
+        description: "The count of cpu cores"
+        example: 4
+      vm_uuid:
+        type: "string"
+        description: "The uuid of virtual machine"
+        example: "9808421e-8426-0dfd-55bd-246c95434e1a"
+      vm_host:
+        type: "string"
+        description: "The host of virtual machine"
+        example: "host.redhat.com"
+      vm_host_socket_count:
+        type: "string"
+        description: "The socket count for the virtual machine host"
+        example: ""
+      vm_host_core_count:
+        type: "string"
+        description: "The core count for the virtual machine host"
+        example: "8"
+      entitlements:
+        type: "array"
+        items:
+          $ref: "#/definitions/SystemEntitlements"
+      is_redhat:
+        type: "boolean"
+        description: "Indicates if the system is a RedHat system"
+      redhat_certs:
+          type: "string"
+          description: "RedHat certificates"
+          example: "183.pem"
+      redhat_package_count:
+          type: "integer"
+          format: "int64"
+          description: "RedHat package count"
+          example: 65 
+  Facts:
+    title: Host facts 
+    description: Facts of a host 
+    type: object
+    properties:
+      namespace:
+        type: "string"
+        description: "Namespace (defaults to 'qpc')"
+        example: "qpc"
+      facts:
+        $ref: "#/definitions/NestedFacts"
+      rh_product_certs:
+        type: array
+        items:
+          type: string
+        description: "List of Red Hat product certificates"
+        example: ""
+      rh_products_installed:
+        type: array
+        items:
+          type: string
+        description: "List of Red Hat products installed"
+        example: [
+          "RHEL"
+        ]
+  SystemProfile:
+      title: System profile fields
+      description: Representation of the system profile fields
+      type: object
+      properties:
+        number_of_cpus:
+          type: integer
+        number_of_sockets:
+          type: integer
+        cores_per_socket:
+          type: integer
+        infrastructure_type:
+          type: string
+        infrastructure_vendor:
+          type: string
+        os_release:
+          type: string
+        os_kernel_version:
+          type: string
+        architecture:
+          type: string
   Host:
-    type: "object"
-    description: "Host data should conform to the host inventory swagger definition described here: https://github.com/RedHatInsights/insights-host-inventory/blob/system_profile_changes/swagger/api.spec.yaml"
+    title: Host data
+    description: >-
+      Data of a single host belonging to an account. Represents the hosts
+      without its Inventory metadata.
+    type: object
+    properties:
```

Comment:
Instead of updating the properties - we need to update the example and remove these. We want this to be an empty object with just an example. 

Also - remove the facts properties/nested facts properties